packages/@aws-cdk/aws-lambda/lib/runtime.ts

```diff
@@ -154,10 +156,19 @@ export class Runtime {
    */
   public readonly family?: RuntimeFamily;
 
+  /**
+   * The bundling Docker image for this runtime.
+   * Points to the lambci/lambda build image for this runtime.
+   *
+   * @see https://hub.docker.com/r/lambci/lambda/
+   */
+  public readonly bundlingDockerImage: assets.BundlingDockerImage;
```

Comment:
I'm not so sure the official AWS CDK should call a third party docker image the lambda runtime's image. I'd rather us drop this and let the user pick.

Reply:
This is the same image used by `sam build`. I think it's totally fine to reference it.